lang/en/docs/models-directory/machine-learning/accuracy.md

```diff
@@ -0,0 +1,32 @@
+# Accuracy
+
+## Importance of Accurate Training Data
+
+When training a machine learning model, care must be taken to ensure that the dataset being used is a quality one.
+Indeed, the concept of "garbage-in, garbage-out" is highly applicable to models trained on poor a low-quality dataset -
+a model is only as good as the data it was trained on.
+
+Consider a hypothetical machine learning model which is able to achieve perfect accuracy on any dataset it's given,
+without overfitting. Now consider what would happen if such a model were trained on two different datasets of vastly
+different quality - for example, electronic energies calculated using
+the [Local-Density Approximation](https://en.wikipedia.org/wiki/Local-density_approximation) versus electronic energies
+calculated with a [Coupled-Cluster](https://en.wikipedia.org/wiki/Coupled_cluster) technique. Despite performing
```

Comment:
Lets not use a direct link but instead add a links section below like in a manuscript

Reply:
Links section added